src/hotspot/share/gc/shenandoah/mode/shenandoahGenerationalMode.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019, 2020, Red Hat, Inc. All rights reserved.
```

Comment:
I believe "Copyright (c) 2020 ..." would be correct there.

Reply:
Fixed.